src/Symfony/Component/AssetMapper/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ CHANGELOG
  * Add a `importmap:audit` command to check for security vulnerability advisories in dependencies
  * Add a `importmap:outdated` command to check for outdated packages
  * Change the polyfill used for the importmap renderer from a URL to an entry in the importmap
+ * Fix exception if assets directory is missing in production environment
```

Comment:
should be removed, this file is only for new features (changelog for bug fixes is generated automatically)

Reply:
Removed!